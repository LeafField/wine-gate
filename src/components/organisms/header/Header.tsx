import React from "react";
import { KumarOne } from "../../../utils/font";
import { Symbol } from "../../atoms/Symbol/Symbol";

const HeaderTitle = () => {
  return (
    <header className="relative flex h-[clamp(3.44rem,9.24vw,6rem)] flex-col items-center justify-center bg-red text-white">
      <p className="text-center text-header-subtitle">
        気軽にワイン入門、気楽にワイン紹介
      </p>
      <h1 className={`${KumarOne.className} text-center text-header-title `}>
        WINE GATE
      </h1>
      <Symbol />
    </header>
  );
};

export default HeaderTitle;
