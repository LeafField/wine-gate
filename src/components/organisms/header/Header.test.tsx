import { screen, render } from "@testing-library/react";
import Header from "./Header";

describe("Headerの結合テスト", () => {
  test("スナップショット", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
