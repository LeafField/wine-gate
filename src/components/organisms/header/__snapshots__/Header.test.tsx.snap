// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTitleの単体テスト スナップショット 1`] = `
<div>
  <header
    class="relative flex h-[clamp(3.44rem,9.24vw,6rem)] flex-col items-center justify-center bg-red text-white"
  >
    <p
      class="text-center text-header-subtitle"
    >
      気軽にワイン入門、気楽にワイン紹介
    </p>
    <h1
      class="className text-center text-header-title "
    >
      <a
        href="/"
      >
        WINE GATE
      </a>
    </h1>
    <figure
      class="symbol absolute inset-0 z-50"
    >
      <img
        alt="wine gate"
        data-nimg="1"
        decoding="async"
        fetchpriority="high"
        height="124"
        src="/symbol.svg"
        style="color: transparent; object-fit: cover;"
        width="124"
      />
    </figure>
  </header>
</div>
`;
